QRCoder/PdfByteQRCode.cs

```diff
@@ -204,11 +204,11 @@ private string CreatePathFromModules()
 
                 // Create a single rectangle for the entire run of dark modules
                 // Format: x y width height re
-                pathCommands.Append(ToStr(startX));
+                pathCommands.AppendInvariant(startX);
                 pathCommands.Append(' ');
-                pathCommands.Append(ToStr(y));
+                pathCommands.AppendInvariant(y);
                 pathCommands.Append(' ');
-                pathCommands.Append(ToStr(x - startX));
+                pathCommands.AppendInvariant(x - startX);
                 pathCommands.Append(" 1 re\r\n");
```

Comment:
Hopefully for v2 this can all be cleaned up into an interpolated string, fully optimized for .NET Standard 2.1+, but compatible with .NET Standard 2.0 as well.